infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/server/OpenShiftServerExposer.java

```diff
@@ -91,95 +87,22 @@
  */
 public class OpenShiftServerExposer extends KubernetesServerExposer<OpenShiftEnvironment> {
 
-  private final OpenShiftEnvironment openShiftEnvironment;
-
   public OpenShiftServerExposer(
-      String machineName, Pod pod, Container container, OpenShiftEnvironment openShiftEnvironment) {
-    super(Collections.emptyMap(), machineName, pod, container, openShiftEnvironment);
-    this.openShiftEnvironment = openShiftEnvironment;
+      ExternalServerExposerStrategy OpenshiftExternalServerExposerStrategy,
+      String machineName,
+      Pod pod,
+      Container container,
+      OpenShiftEnvironment kubernetesEnvironment) {
+    super(
+        OpenshiftExternalServerExposerStrategy, machineName, pod, container, kubernetesEnvironment);
   }
 
   @Override
   protected void exposeExternalServers(
```

Comment:
Why do we need still override this method since it's the same as KubernetesServerExposer's implementation?